### ISSUE TITLE
Large correlation tables and CIs

### ISSUE THREAD

**Issue by bwiernik** [dstanley4/apaTables#25]
I am quite in favor of always reporting confidence intervals with the package functions. However, when a correlation matrix is very large, the number of intervals can become unwieldy and unhelpful. Often, all of the variables are included in such a table for completeness in reporting and not because they form the basis for direct inference (e.g. the correlations among 15 indicators at 4 time points). 

For such cases, it would be useful to provide options to summarize CI information or limit the CIs to correlations of interest. For example, `show.conf.interval = "note"` could provide an approximate summary confidence interval in the table note: “Correlations have a confidence interval margin of error ≈ +/– .12.” `show.conf.interval` could perhaps also take a named vector of the variables of interest for which to report CIs (e.g., interest might be in correlations with a total score, but facet scales are reported for completeness as well; demographic correlations might be reported for completeness, but not be of substantive interest). 

In either of these cases, to discourage dichotomous interpretations, I would say to omit significance stars when CIs are suppressed.

**Comment by gserapio** [comment]
Yes, in the cases of large correlation matrices it's extremely time consuming to remove all of the CIs manually from a word doc! This feature would be a lifesaver.

**Comment by awmeade** [comment]
Agreed.  Having CI on as the default would be a lot more user friendly than forcing it to be on.
